js/tests/unit/tab.spec.js

```diff
@@ -548,6 +548,72 @@ describe('Tab', () => {
       expect(Event.prototype.stopPropagation).toHaveBeenCalledTimes(2)
       expect(Event.prototype.preventDefault).toHaveBeenCalledTimes(2)
     })
+
+    it('if keydown event is right arrow and next element is disabled', () => {
```

Comment:
Should we introduce tests for all keys (up and down arrows) -> to Bs

Reply:
We decided by GeoSot not to include them. We want only to test the `disabled` part here. The fact that right/next = down/next is tested elsewhere in the test file.